packages/react-resizable-panels/src/utils/callPanelCallbacks.ts

```diff
@@ -40,7 +41,8 @@ export function callPanelCallbacks(
       const { onCollapse, onExpand, onResize } = callbacks;
 
       if (onResize) {
-        onResize(mixedSizes, lastNotifiedMixedSizes);
+        // On first render, lastNotifiedMixedSizes will be undefined, fall back to mixedSizes
+        onResize(mixedSizes, lastNotifiedMixedSizes ?? mixedSizes);
```

Comment:
There are no "previous" sizes for the initial render, so this value _should_ be `undefined`.  I think the only thing that should be changed here is the type definition:
```ts
export type PanelOnResize = (
  mixedSizes: MixedSizes,
  prevMixedSizes: MixedSizes | undefined
) => void;
```

Reply:
Thanks, I'll close this PR and open a fresh one with the minimal type change.